contracts/src/v0.8/keystone/test/mocks/CapabilityConfigurationContract.sol

```diff
@@ -10,4 +10,8 @@ contract CapabilityConfigurationContract is ICapabilityConfiguration, ERC165 {
   function getCapabilityConfiguration(uint256 donId) external view returns (bytes memory configuration) {
     return s_donConfiguration[donId];
   }
+
+  function supportsInterface(bytes4 interfaceId) public pure override returns (bool) {
```

Comment:
I don't see this used anywhere, why is it needed?

Reply:
This is called by consumers/callers of this contract. My guess is the capability registry will call this prior to adding a particular configuration contract address to check that it supports the interface.

Reply:
It is used here: https://github.com/smartcontractkit/chainlink/pull/13031/files#diff-d94c1796b3544ef8780af7e16b5361c8fb9e4d63f492faeda6a3362a8363f51bR180-R182 (previous implementation and test weren't working as expected).